src/docs/truffle/reference/configuration.md

```diff
@@ -315,10 +315,11 @@ In the `compilers` object you can specify settings related to the compilers used
 Solidity compiler settings. Supports optimizer settings for `solc`, as well as other settings such as debug and metadata settings.
 
 You may specify...
-+ any solc-js version listed at [solc-bin](http://solc-bin.ethereum.org/bin/list.json). Specify the one you want and Truffle will get it for you.
-+ a natively compiled solc binary (you'll need to install this yourself, links to help below).
-+ a dockerized solc from one of images published [here](https://hub.docker.com/r/ethereum/solc/tags/).
++ any solc-js version (using semver) listed at [solc-bin](http://solc-bin.ethereum.org/bin/list.json). Specify the one you want and Truffle will get it for you.
++ "native" to use a natively compiled solc binary (you'll need to install this yourself, links to help below).
++ a dockerized solc tag from one of images published [here](https://hub.docker.com/r/ethereum/solc/tags/).
 + a path to a locally available solc
```

Comment:
IMO this should be clarified to state that this means a path to a locally available soljson, not a solc binary... but also I'm OK with not doing that in this PR and doing that separately later.

Reply:
agree!